src/pages/billing/Project.js

```diff
@@ -60,7 +66,7 @@ const ExpandedInfoRow = Utils.memoWithName('ExpandedInfoRow', ({ title, details,
   ])
 })
 
-const WorkspaceCard = Utils.memoWithName('WorkspaceCard', ({ workspace, isExpanded, onExpand }) => {
+const WorkspaceCard = Utils.memoWithName('WorkspaceCard', ({ workspace, statusIcon, isExpanded, onExpand }) => {
```

Comment:
It looks like we have the information we need to derive the `statusIcon` in this component without having to pass in additional props.
```suggestion
const WorkspaceCard = Utils.memoWithName('WorkspaceCard', ({ workspace, isExpanded, onExpand }) => {
```

Reply:
Not exactly. We need to know if all the workspaces have the right billing account, not just if the current one has it. This function lacks the context to know that.